src/docfx/build/xref/InternalXrefMap.cs

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    internal class InternalXrefMap
```

Comment:
>InternalXrefMap [](start = 19, length = 15)

what's "Internal" meaning?

Reply:
There are 2 pars of xref map, internal is the part defined within the docset. The other part is restoring external xref map files. Both of them would be used for resolving.